src/coreclr/.nuget/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.pkgproj

```diff
@@ -0,0 +1,24 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <PackageDescription>This package provides a low-level ReadyToRun file format decoder.</PackageDescription>
```

Comment:
```suggestion
    <PackageDescription>This package provides a low-level ReadyToRun file format decoder. This package is experimental.</PackageDescription>
```

I want to make sure we put police tape around this so that we don't accidentally end up with a "officially" looking Microsoft-signed binary on NuGet.org that didn't actually go through our usual quality process. There's a risk a package like this could be use in security-sensitive context (malware researchers) and we don't provide any security guarantees (we don't review or fuzz this). This code was written in the context of trusted inputs.

Maybe we should put Experimental in the package name too. It should be clear there are potential dragons here.